src/components/Header/Header.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+
+type Props = {
+  handleSubmit: (
+    event: React.FormEvent<HTMLFormElement>,
+    todoTitle: string,
+    setTodoTitle: (title: string) => void,
+  ) => void;
+};
+
+export const Header: React.FC<Props> = ({ handleSubmit }) => {
+  const [todoTitle, setTodoTitle] = useState('');
+
+  return (
+    <header className="header">
+      <h1>todos</h1>
+
+      <form onSubmit={(e) => handleSubmit(e, todoTitle, setTodoTitle)}>
```

Comment:
we can only send todoTitle in handleSubmit, let's create some handler, and use `e.preventDefault` there and the same with resetting todoTitle